ui/admin/settings-reset.php

```diff
@@ -106,15 +106,20 @@
 ?>
     <hr />
 
-    <h3>Reset Weekend</h3>
+    <h3><?php _e( 'Reset Weekend', 'pods' ); ?></h3>
 
-    <p>This feature has been exclusively built for Pods to help developers suffering from "Monday", and allows them to reset the weekend.</p>
-    <p>By resetting the weekend, you will be sent back to Friday night and the weekend you've just spent will be erased. You will retain all of your memories of the weekend, and be able to relive it in any way you wish.</p>
+    <p><?php _e( 'This feature has been exclusively built for Pods to help developers suffering from "Monday", and allows them to reset the weekend.', 'pods' ); ?></p>
+    <p><?php _e( "By resetting the weekend, you will be sent back to Friday night and the weekend you've just spent will be erased. You will retain all of your memories of the weekend, and be able to relive it in any way you wish.", 'pods'); ?></p>
 
     <p class="submit">
+<<<<<<< HEAD
```

Comment:
I had to revert, didn't notice it but there's some git merge text leftover here you'll want to address.